README.md

```diff
@@ -200,4 +201,14 @@ NULL ...
 ### Internals
 ![](./assets/internals.png)
 
+## Versioning
+The release version of django-spanner in [setup.py](./setup.py) must follow this form
+```python
+version = '<DJANGO_VERSION><RELEASE_MODE><RELEASE_VERSION>'
+```
+for example, if this package supports Django version 2.2.X, and we are in alpha release mode, while this application's release is version 0, our version will be
+```python
+version = '2.2a0'
```

Comment:
a0 means "pre-alpha" (never released). For the first alpha release, it should be a1.